kinesis-s3/start.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Source library
+. ../utils/helper.sh
+
+source config/demo.cfg
+
+check_env || exit 1
+check_running_cp 5.2 || exit
+check_aws || exit
+
+./stop.sh
+
+# Install Connectors and start Confluent Platform
+confluent-hub install confluentinc/kafka-connect-kinesis:latest --no-prompt
+confluent-hub install confluentinc/kafka-connect-s3:latest --no-prompt
+
+#---------------------------------
+# Option 1: Confluent Cloud SR
+#SCHEMA_REGISTRY_CONFIG_FILE=$HOME/.ccloud/config
+
+# Option 2: local Confluent SR
+SCHEMA_REGISTRY_CONFIG_FILE=schema_registry.config
+confluent start schema-registry
+#---------------------------------
+
+# Generate CCloud configurations
+../ccloud/ccloud-generate-cp-configs.sh $SCHEMA_REGISTRY_CONFIG_FILE
+CONFLUENT_CURRENT=`confluent current | tail -1`
+DELTA_CONFIGS_DIR=delta_configs
+source $DELTA_CONFIGS_DIR/env.delta
+
+# Start Connect that connects to CCloud cluster
+mkdir -p $CONFLUENT_CURRENT/connect
+CONNECT_CONFIG=$CONFLUENT_CURRENT/connect/connect-ccloud.properties
+cp $CONFLUENT_HOME/etc/schema-registry/connect-avro-distributed.properties $CONNECT_CONFIG
+cat $DELTA_CONFIGS_DIR/connect-ccloud.delta >> $CONNECT_CONFIG
+CONNECT_REST_PORT=8087
+cat <<EOF >> $CONNECT_CONFIG
+rest.port=$CONNECT_REST_PORT
+rest.advertised.name=connect-cloud
+rest.hostname=connect-cloud
+group.id=connect-cloud
+request.timeout.ms=20000
+retry.backoff.ms=500
+plugin.path=$CONFLUENT_HOME/share/java,$CONFLUENT_HOME/share/confluent-hub-components
+EOF
+export CLASSPATH=$CONFLUENT_HOME/kafka-connect-replicator/kafka-connect-replicator-5.2.1.jar && connect-distributed $CONNECT_CONFIG > $CONFLUENT_CURRENT/connect/connect-ccloud.stdout 2>&1 &
```

Comment:
Why is Replicator jar being added to the classpath in this demo? Doesn't seem like it's needed here, but maybe I'm missing something

Reply:
@mikebin -- at the time I added it because otherwise it was generating an error, but I can't reproduce it.  The script has evolved quite a bit and now seems to work with out it.  I'll remove it since it appears to work without it now.  Thanks for mentioning this!